.circleci/config.yml

```diff
@@ -1,67 +1,33 @@
 version: 2.1
-workflows:
-  build:
-    jobs:
-      - sous-chefs/lint:
-          name: Lint
-      - sous-chefs/dokken:
-          name: default
-          suite: default
-          requires:
-            - Lint
 
 orbs:
-  sous-chefs:
-    executors:
-      ruby:
-        docker: [ image: circleci/ruby ]
-      chef:
-        machine: true
+  kitchen: sous-chefs/kitchen@1.0.0
+
+workflows:
+  kitchen:
     jobs:
-      lint:
-        executor: ruby
-        parameters:
-          channel:
-            description: ChefDK channel stable or current
-            type: string
-            default: current
-        steps:
-          - checkout
-          - run:
-              name: Install Chef
-              command: curl -L https://omnitruck.chef.io/install.sh | sudo bash -s -- -c <<parameters.channel>> -P chefdk
-          - run:
-              name: Unit Test
-              command: chef exec delivery local all
-      dokken:
-        executor: chef
-        parameters:
-          suite:
-            description: Test Kitchen suite name
-            type: string
-          channel:
-            description: ChefDK channel stable or current
-            type: string
-            default: current
-        environment:
-          KITCHEN_LOCAL_YAML: .kitchen.dokken.yml
-        steps:
-          - checkout
-          - run:
-              name: Launch Haveged container for entropy
-              command: docker run --privileged -d harbur/haveged:1.7c-1
```

Comment:
If this is needed it should be in the recipe, not in the dokken config.

Reply:
(that way it works for dokken and kitchen)

Reply:
Well it's not 100% necessarily to have it in the recipe. Example I'm running this cookbook on a physical node, it has enough entropy for gpg key generation to be super fast.
So if it is in the recipe, it should be optional.

Reply:
We can put it in the test recipe, so it will only run on Tests, when gpg is released we will swap over to resource driven,

also note to @damacus we seem to use different gpg generation config to the cookbook, as I do not use gpg can you take a look and see if it makes any different?